common/scala/src/main/scala/org/apache/openwhisk/core/entity/WhiskActivation.scala

```diff
@@ -137,6 +137,7 @@ object WhiskActivation
 
   /** Some field names for annotations */
   val pathAnnotation = "path"
+  val bindingAnnotation = "binding"
```

Comment:
will check down below but this raises the opportunity to have the name in the activation be that of the binding, and the annotation is that of the package.

Reply:
I'm sorry, I don't understand this. 

In the example below, I think the name `triple` in the activation can not be the name of the binding `whisk.system/p2`

- there is an action `whisk.system/pkg1/triple`
- and the `whisk.system/p2` is binding the `whisk.system/pkg1`
- and i invoked an action `whisk.system/p2/triple`

```json
{
    "namespace": "whisk.system",
    "name": "triple",
    "version": "0.0.1",
    "subject": "whisk.system",
    "activationId": "b86606b16b8a4371a606b16b8a437182",
    "start": 1552900408086,
    "end": 1552900408090,
    "duration": 4,
    "annotations": [
        {
            "key": "binding",
            "value": "whisk.system/p2"
        },
        {
            "key": "path",
            "value": "whisk.system/pkg1/triple"
        },
...
```